### DISCUSSION TITLE
What am I missing???

### DISCUSSION THREAD

**Discussion by Lactionaire** [tagyoureit/nodejs-poolController#983] [Q&A] [answered]
I can't seem to get any readings in the control panel.  Per the comms tab I am getting 100% success rate of bytes sent however nothing is showing in the panel.  I have a new WhisperFlo VST Pentair pump with the data cable hardwired to a RS485 usb receiver in a windows PC.  On the display of the physical pump when I turn on the service it says AUTO on the pump so its clearly knows its hooked to something.  When I put the software into service mode all functions are then working on the pump.

See screenshots:

![Screenshot 2024-06-25 092740](https://github.com/tagyoureit/nodejs-poolController/assets/66134126/d422282d-c54e-4ed5-a9ca-77f22298274c)
![Screenshot 2024-06-25 092708](https://github.com/tagyoureit/nodejs-poolController/assets/66134126/1ecd0ee2-50b9-4407-bd0f-411d4b859c5b)

What am I missing?  Thanks.

**Comment by tagyoureit** [comment]
Under the Config->Pumps tab you need to add circuits for the speeds.  You should start with the Pool circuit and then can add more for different speeds.

**Comment by Lactionaire** [comment]
![Screenshot 2024-06-25 134055](https://github.com/tagyoureit/nodejs-poolController/assets/66134126/262ec72a-549b-4ead-a126-2839d468fd76)

Yeah, I had already had those set up, but still nothing.  I would think it would also show that the pump is on or something to that effect.

**Reply by tagyoureit** [reply, marked as answer]
Pump Speed +/- are specific for spa side remotes.  You'll need to setup a new aux circuit for the _FASTER_ speed and set the Pool circuit to be the slower speed.

**Comment by Lactionaire** [comment]
That worked, thank you.
